lib/percona_migrator/runner.rb

```diff
@@ -0,0 +1,51 @@
+require 'open3'
+
+module PerconaMigrator
+  class Runner
+
+    NONE = "\e[0m"
+    CYAN = "\e[38;5;86m"
+    GREEN = "\e[32m"
+    RED = "\e[31m"
```

Comment:
I wanted to wait to add a new dependency until it was necessary. I had https://github.com/peter-murach/pastel in mind but looks like colorize is the standard.